django/thunderstore/social/api/experimental/views/current_user.py

```diff
@@ -25,7 +27,7 @@ def get(self, request, format=None):
 
 
 class SubscriptionStatus(TypedDict):
-    expires: Optional[date]
+    expires: Optional[datetime.date]
```

Comment:
This type is wrong for what it's worth, since you're assigning a full timestamp to it (meaning it should be of type `datetime.datetime`)